GB_baseSwift/main.swift

```diff
@@ -7,5 +7,81 @@
 
 import Foundation
 
+enum EngineStatus {
+    case launched
+    case drownedOut
+}
 
-// Here will be your code. It is a developer branch
+enum WindowsStatus {
+    case opened
+    case closed
+}
+
+struct LiteCar {
+    let carModel: String
+    let yearOfIssue: Int
+    let trunkVolume: Int
+    var carEngine: EngineStatus {
+        willSet {
+            print ("Engine will be \(newValue) in 1 second")
+        }
+    }
+//  init (carEngine: EngineStatus = .drownedOut) {
+//      self.carEngine = carEngine
+//  }
+// Если вставить эту часть, то он считает, что все остальные параметры не нужны и пишет ошибку в 57 строчке "Cannot infer contextual base in reference to member 'Closed'"
```

Comment:
https://stackoverflow.com/questions/67494326/swiftui-cannot-infer-contextual-base-in-reference-to-member